collections/guides/quickstarts/quickstart_dotnet.md

```diff
@@ -0,0 +1,243 @@
+---
+layout: default
+title: Quickstart with .NET
+permalink: /getting_started/quickstart_with_dotnet/
+categories: quickstart
+nav_order: 1
+has_children: true
+has_toc: false
+description: Get Started with Basis Theory with .NET
+image:
+    path: https://cdn.basistheory.com/images/seo/guides-opengraph.png
+    width: 1200
+    height: 630
+---
+# Quickstart with .NET
+
+In this tutorial, you will create a new .NET console application and use Basis Theory’s platform to secure a string of data and then read that data back out. Once you’ve completed this tutorial, you’ll have a foundational understanding of how to secure your data with Basis Theory allowing you to no longer worry about the underlying storage or encryption of the data.
+
+If you are already familiar with Basis Theory and our platform in general, our guide on [How to send Token data via HTTP](https://guides.basistheory.com/guides/use-token-data-in-http-requests/) or [Collecting payments with Elements](https://guides.basistheory.com/guides/collect-atomic-cards-with-elements/) may be a better place to start.
+
+## Step 1: Get your .NET environment ready
+
+### Step 1.1: Ensure you have .NET installed
+
+Completing this tutorial will require you to have the .NET 6.0 SDK or later installed on your local system. The latest .NET SDK can be [downloaded here](https://dotnet.microsoft.com/en-us/download/dotnet).
+
+### Step 1.2: Create a .NET console application
+To start, you will need a new .NET console application. Create a new console application in your IDE of choice named `BasisTheoryQuickstart` or use the following command to create the application:
+
+```bash
+dotnet new console -n BasisTheoryQuickstart -o BasisTheoryQuickstart
+```
+
+### Step 2: Import needed dependencies
+
+The simplest form of securing data with Basis Theory is through our API with a JSON body. In .NET, you can start with just an HttpClient, for this example we use the `System.Net.Http` and `System.Text.Json` assemblies to make an API call. At the top of your file, you’ll import the following dependencies:
+
+```csharp
+using System.Net.Http.Json;
+using System.Text.Json;
+```
+
+## Step 3: Create a new Application in your Basis Theory Account
+
+If you don’t already have a Basis Theory account and your first Tenant, create one [here](https://portal.basistheory.com/register).
+
+Within your Basis Theory account, create a new Application. To create a new application, head to our portal [here](https://portal.basistheory.com/applications/create) — this Application’s API Key will enable you to authenticate with the Basis Theory platform and create Tokens within your Tenant.
+
+### 3.1 Enter an Application Name
+This name allows you to identify this application in the future — for this tutorial enter “Ruby Quickstart”.
```

Comment:
"Ruby Quickstart" should probably be ".NET Quickstart" 😜

Reply:
Fixed